actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1222,7 +1222,7 @@ def hidden_field(object_name, method, options = {})
       # * Creates standard HTML attributes for the tag.
       # * <tt>:disabled</tt> - If set to true, the user will not be able to use this input.
       # * <tt>:multiple</tt> - If set to true, *in most updated browsers* the user will be allowed to select multiple files.
-      # * <tt>:include_hidden</tt> - When <tt>multiple: true</tt> and <tt>include_hidden: true</tt>, the field will be prefixed with an <tt><input type="hidden"></tt> field with an empty value to support submitting an empty collection of files.
+      # * <tt>:include_hidden</tt> - When <tt>multiple: true</tt> and <tt>include_hidden: false</tt>, the field will be prefixed with an <tt><input type="hidden"></tt> field with an empty value to support submitting an empty collection of files.
```

Comment:
I don't think this comment is quite right. When using `multiple: true` and `include_hidden: true`, then the field _will_ get the `<input type="hidden">` which enables submitting an empty collection.

If I understand the other issue correctly, the issue is that `include_hidden` is _defaulting_ to `true` because of `config.active_storage.multiple_file_field_include_hidden` and the docs don't make it clear that you may want to change that. I think the correct documentation change would be to add a new line describing that the default value may be `true` and why you would want to change it to `false`.